controllers/meetings.js

```diff
@@ -66,48 +60,33 @@ exports.create = function(req, res) {
  * Shows ONE meeting
  */
 exports.read = function(req, res) {
-	Meeting.findById(req.params.meeting_id).populate('admin participants', 'name -_id').populate('availability', 'username').exec(function(err, meeting) {
+	Meeting.findById(req.params.meeting_id).populate('admin participants', 'name -_id').populate('availability').exec(function(err, meeting) {
 		if (err) {
 			return res.sendStatus(404);
 		}
 		if (!meeting) {
 			return res.sendStatus(404).send("This meeting does not exist.");
 		} else {
-			var o = {
-				name: meeting.name,
-				admin: meeting.admin,
-				description: meeting.description,
-				date: meeting.date,
-				participants: meeting.participants,
-				lastUpdated: meeting.lastUpdated,
-				availability: meeting.avail
-			};
-			res.json(o);
+			res.json(meeting);
 		}
 	});
 };
-
 // Can also use findByIdAndUpdate  ...
 exports.update = function(req, res) {
 	// update the meeting object
 	Meeting.findById(req.params.meeting_id, function(err, meeting) {
 		if (err) {
-			res.send(404);
+			return res.send(404);
 		}
-		meeting = _.assign(meeting, req.body);
-		meeting.lastUpdated = Date.now();
+          meeting.name = (req.body.name);
+          meeting.description= (req.body.description);
+          meeting.date = (req.body.date);
+		  meeting.lastUpdated = Date.now();
 		meeting.save(function(err) {
 			if (err) {
-				return res.status(400);
+				return res.send(err);
```

Comment:
updated I had changed it to log an error since the 400 status code was vague forgot to change it back and 500 would be a better status coode